src/components/ClientsId.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { FC, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { Typography } from '@mui/material';
+import PostService from '../api/PostServise';
+import { useFetching } from '../hooks/useFetching';
+import { getClientId } from '../store/clients/actions';
+
+interface Props {
+  clientID: number | null;
+}
+
+export const ClientsId: FC<Props> = ({ clientID }) => {
+  const dispatch = useDispatch();
+
+  const [fetchClientsById] = useFetching(async (id: number) => {
+    const response = await PostService.getClientId(id);
+    dispatch(getClientId(response.data));
+  });
+
+  useEffect(() => {
+    if (clientID === null) return;
```

Comment:
Create a separate variable for  `clientID === null`. Like this
`const isNoClientID = clientID === null`

Reply:
Done